### PR TITLE
Added OSS cluster tests

### PR DESCRIPTION
**Pull request by vladvildanov**
Tests to prove that Relay connection can be used within cluster(aggregate) connection

### CODE REVIEW

**Comment by vladvildanov** [comment]
@tillkruss Do you think we need more cluster test cases?

File: src/Connection/RelayConnection.php [hunk 1]
```diff
@@ -124,14 +124,6 @@ public function getClient()
         return $this->client;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function getIdentifier()
-    {
-        return $this->client->endpointId();
```
Comment (vladvildanov) [inline src/Connection/RelayConnection.php:132]: In the context of connection this method doesn't need at all, cause it's just used to get a unique identifier for connection used on the client side. Here in RelayConnection this method requires connection to server, before actual connection is established.

**Comment by tillkruss** [comment]
> @tillkruss Do you think we need more cluster test cases?

No, I think having minimal tests for Cluster is fine, it's an edge-case IMO.

**Review by tillkruss — APPROVED** [review]
